packages/components/atoms/f-form-field/src/components/FormFieldAffixed.vue

```diff
@@ -0,0 +1,189 @@
+<template>
+    <div
+        :class="[
+            $style['c-formField'],
+            $style['c-formField--affixed'],
+            $style[`c-formField--affixed--${fieldSize}`], {
+                [$style['c-formField--invalid']]: hasError,
+                [$style['c-formField--disabled']]: isDisabled
+            }
+        ]"
+        :data-test-id="testId.affixed">
+        <input
+            :id="$attrs.id"
+            ref="input"
+            :class="$style['c-formField-field']"
+            :disabled="attributes.disabled"
+            v-bind="attributes"
+            :value="value"
+            :data-test-id="testId.input"
+            @change="updateInput">
+
+        <div
```

Comment:
Should these be buttons elements?

Reply:
These aren't interactive – just visual prefix/suffix elements that display on top of the form field

Reply:
would they not be better as a `span` ?